wof/wofpy_config.py

```diff
@@ -14,6 +16,7 @@
 
 Usage:
     wofpy_config INDIR
+    wofpy_config INDIR [--mode=<test,production>]
```

Comment:
@ocefpaf yea. now that I think about it.. it does seems weird. Maybe `development` is better? What do you think @emiliom